### DISCUSSION TITLE
Expo Compatibilty

### DISCUSSION THREAD

**Discussion by bwjohns4** [microsoft/react-native-windows#12320] [General]
Is this compatible with Expo Bare (ejected). I can't find any info anywhere on how I could add Windows to my ejected Expo app.

**Comment by Eduard-Hasa** [comment]
bump also wondering this?

**Comment by acoates-ms** [comment]
I've been looking into this.  There are a few modifications needed to expo to support out of tree platforms better.
I'm working on getting these fixes submitted to expo.

**Reply by mataide** [reply]
@acoates-ms Love this as well, I am trying to make it work.

**Reply by sshrshnv** [reply]
@acoates-ms any progress?
